test/integration/scribble/run/valid.spec.ts

```diff
@@ -40,7 +40,7 @@ describe(`Command "scribble <filename>" is working properly`, () => {
 
             it("STDOUT is correct", () => {
                 // Uncomment next line to update instrumented sources
-                // fse.writeFileSync(instrFileName, output, { encoding: "utf-8" });
+                fse.writeFileSync(instrFileName, output, { encoding: "utf-8" });
```

Comment:
I guess this should be commented out.

Reply:
Fixed in [029c9c3](https://github.com/ConsenSys/scribble/pull/211/commits/029c9c3117655fdbc0f0f348bcd11e354e5ce2ad)